service/dap/server.go

```diff
@@ -1049,6 +1071,32 @@ func (s *Session) onLaunchRequest(request *dap.LaunchRequest) {
 		return
 	}
 
+	func() {
```

Comment:
Why?

Reply:
If you refer to this file, that pattern is to execute a defer, not to group anything.